IPA.Loader/Config/ConfigRuntime.cs

```diff
@@ -31,7 +26,12 @@ public int GetHashCode(DirectoryInfo obj)
         }
 
         private static readonly ConcurrentBag<Config> configs = new ConcurrentBag<Config>();
-        private static readonly AutoResetEvent configsChangedWatcher = new AutoResetEvent(false);
+        private static readonly Action configsChangedAction = () =>
+        {
+            foreach (var config in configs.Where(c => c.Store != null).ToArray())
+                config.Store.SyncAction = () => RequiresSave.Add(() => Save(config));
```

Comment:
Note here,
`() => RequiresSave.Add()` is the outer action and
`() => Save(config)` is the inner action that is going to be processed by the save thread